### DISCUSSION TITLE
Pls include example on function calling (LLM) as client

### DISCUSSION THREAD

**Discussion by pervrosen** [modelcontextprotocol/.github#22] [Q&A] [answered]
by using an open source LLM and a whatever glue is needed to connect to eg the Weather Server. 

This looks very promising indeed! Thx! //PvR

**Comment by dsp-ant** [comment, marked as answer]
MCP is a protocol between a client and a server. In this case any application using an open source model would have to implement MCP so servers can connect. The SDKs and respective client APIs, as well as the spec documentation are the best place to look for how to implement clients.
